interfaces/builtin/block_devices.go

```diff
@@ -66,6 +66,7 @@ const blockDevicesConnectedPlugAppArmor = `
 /dev/i2o/hdd[a-x] rwk,                                     # I2O hard disk continued
 /dev/mmcblk[0-9]{,[0-9],[0-9][0-9]} rwk,                   # MMC (up to 1000 devices)
 /dev/vd[a-z] rwk,                                          # virtio
+/dev/loop[0-9]{,[0-9],[0-9][0-9]} rwk,                     # loopback (up to 1000 devices)
```

Comment:
what about loop-control itself?

Reply:
Good point, ideally /dev/loop-control should be accessible as well